src/icons/delete.vue

```diff
@@ -0,0 +1,13 @@
+<template lang="">
```

Comment:
```suggestion
<template>
```
remove empty `lang` attribute everywhere